xena/tests/test_single_meeting_settings_changes_0.py

```diff
@@ -220,7 +220,7 @@ def test_home_page(self):
         self.jobs_page.log_out()
         self.course_page.hit_url(self.term.id, self.section.ccn)
         self.login_page.dev_auth(self.instructor.uid)
-        self.course_page.wait_for_diablo_title(f'{self.section.code}, {self.section.number}')
+        self.course_page.wait_for_diablo_title('Welcome')
```

Comment:
@mohamalnadi I think this indicates a regression in the diablo code - the page title should have the section code and number instead of the default "Welcome". You could just fix it in your next PR or open a Jira for it.